tests/monitors/kubernetes_scheduler/kubernetes_scheduler_test.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+
+from tests.helpers.metadata import Metadata
+from tests.helpers.verify import verify
+
+pytestmark = [pytest.mark.kubernetes]
+METADATA = Metadata.from_package("kubernetes/scheduler")
+
+
+@pytest.mark.skip("only works with real minikube currently")
```

Comment:
This can conditionalize the skip based on whether the `--no-use-minikube` option was passed.

Reply:
That text isn't quite accurate. The real problem is that the test only works with 1.15 since new metrics were added. Going to add logic to only run for 1.15 or maybe just latest.